fsw/cfe-core/src/inc/cfe_sb.h

```diff
@@ -55,6 +55,15 @@
 
 #define CFE_SB_INVALID_MSG_ID           0xFFFF /**< \brief Initializer for #CFE_SB_MsgId_t values that will not match any real MsgId */
 
+/**
+ * \defgroup CFESBPktTypeDefs cFE SB Packet Type Defines
+ * \{
+ */
+#define CFE_SB_PKTTYPE_CMD              1      /**< \brief #CFE_SB_GetPktType response for command packets */
+#define CFE_SB_PKTTYPE_TLM              0      /**< \brief #CFE_SB_GetPktType response for telemetry packets */
+#define CFE_SB_PKTTYPE_INVALID          0xFF   /**< \brief #CFE_SB_GetPktType response if message type can not be determined */
```

Comment:
As mentioned in CCB - For enumerated values like this we should keep zero as the reserved/invalid value (e.g. in case one of these values is stored in memory which is memset to zero, that should alias to `INVALID`, not `TLM`.